deployments/docker-compose-example/.env.sample

```
@@ -12,13 +12,14 @@ SILL_GITHUB_TOKEN=xxxxx
 SILL_API_PORT=3084
 SILL_IS_DEV_ENVIRONNEMENT=true
 SILL_EXTERNAL_SOFTWARE_DATA_ORIGIN=wikidata
+SILL_IMPORT_WIKIDATA=|
+        { 
+          ids: ['Q10135']
+        }
```

tu veux pas faire une list séparé par des virgules, ça serait beaucoup plus simple et tiendrait sur un ligne :
```suggestion
SILL_IMPORT_WIKIDATA=Q10135,Q10222
```

Et tu n'a plus qu'à faire `SILL_IMPORT_WIKIDATA.split(",")`